include/fplus/pairs.hpp

```diff
@@ -151,6 +151,33 @@ auto zip(const ContainerIn1& xs, const ContainerIn2& ys)
     return zip_with(MakePair, xs, ys);
 }
 
+// API search type: broadcast : (a, [b])  -> ([a], [b])
+// fwd bind count: 1
+// Create a replicate sequence having value x of the same size as ys,
+// and return along with ys as a pair
+// broadcast(5, [1,2,3]) == ([5,5,5], [1,2,3])
+template <typename X, typename ContainerIn,
+        typename Y = typename ContainerIn::value_type,
+        typename ContainerOutX = typename internal::same_cont_new_t<ContainerIn, X>::type,
+        typename ContainerOutY = typename internal::same_cont_new_t<ContainerIn, Y>::type>
+auto broadcast(const X &x, const ContainerIn &ys) {
+    ContainerOutX firsts(ys.size(), x);
+    ContainerOutY seconds{ys.begin(), ys.end()};
+    return std::make_pair(firsts, seconds);
+}
+
+// API search type: broadcast : (a, [b])  -> [(a, b)]
```

Comment:
Changing this to

```cpp
// API search type: broadcast : (a, [b])  -> [(a, b)]
```

should fix your issue with the `autogenerated_defines`.

Reply:
Oops. thank you for catching the typo :) 
It turns out the generate result differed b/c I have BSD `sed` as the default one. After replacing it with `gsed` in the script, the generated file looks as expected.

Reply:
Oh. Yes, I have to admit the solution using `sed` might not be ideal. :grimacing: